settei/presets/flask.py

```diff
@@ -7,17 +7,32 @@
 import collections.abc
 import typing
 
+from flask import Flask
 from typeguard import typechecked
 from werkzeug.datastructures import ImmutableDict
 from werkzeug.utils import cached_property
 
 from ..base import config_property
+from ..utils import import_hook
 from .logging import LoggingConfiguration
 
 __all__ = 'WebConfiguration',
 
 
 class WebConfiguration(LoggingConfiguration):
+    """
```

Comment:
```suggestion
    """Example::
        
        ...
```
It is unncessary to write a new line at the start of docstring.

Reply:
I think it is more perfect if there is description for `WebConfiguration`

```suggestion
    """Settei configuration for the Flask_. For more information, See the example below::
    
        ....
    
    .. _Flask:: ???
```